build-system/tasks/dist.js

```diff
@@ -118,6 +118,9 @@ async function runPreDistSteps(watch) {
  * @return {!Promise}
  */
 async function dist() {
+  if (argv.sanatize_for_diff && !argv.pseudo_names) {
+    throw new Error('--sanatize_for_diff requires --pseudo_names');
+  }
```

Comment:
A few requests:

-`s/sanatize/sanitize/`
- copyright 2015 -> 2020
- can we call the flag `—sanitize_variables` for clarity?
- Finally, do you mind moving this to a separate PR? (Happy to do it myself, but I think you ought to retain authorship)

Reply:
https://github.com/ampproject/amphtml/pull/28156